YAXLib/Pooling/YAXLibPools/SerializerPool.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) Sina Iravanian, Julian Verdurmen, axuno gGmbH and other contributors.
+// Licensed under the MIT license.
+
+#nullable enable
+using System;
+using System.Threading;
+using YAXLib.Pooling.SpecializedPools;
+
+namespace YAXLib.Pooling.YAXLibPools;
+
+/// <summary>
+/// The object pool for <see cref="YAXSerializer"/>.
+/// </summary>
+internal sealed class SerializerPool : PoolAbstract<YAXSerializer>
+{
+    private static readonly Lazy<SerializerPool> Lazy = new(() => new SerializerPool(),
+            LazyThreadSafetyMode.PublicationOnly);
+
+    /// <summary>
+    /// CTOR.
+    /// </summary>
+    /// <remarks>
+    /// <see cref="SpecializedPoolAbstract{T}.Policy"/> must be set before initializing the pool
+    /// </remarks>
+    private SerializerPool()
+    {
+        // This means 50 recursive de/serializations for an object, which should usually suffice.
+        // The pool is focusing on the high frequency of creating child serializers,
+        // not a high number of child serializers at a time.
+        Policy.MaximumPoolSize = 50;
```

Comment:
As described in the comment, it looks small. Too small?

Reply:
Would set `SerializerOptions.DefaultMaxRecursion = 50` to prevent running low in memory.